src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.CurlResponseMessage.cs

```diff
@@ -426,10 +431,25 @@ private void CheckDisposed()
                 }
             }
 
-            [Conditional(VerboseDebuggingConditional)]
-            private void VerboseTrace(string text = null, [CallerMemberName] string memberName = null)
+            private void EventSourceTrace<T>(string formatMessage, T arg0, [CallerMemberName] string memberName = null)
+            {
+                if (EventSourceTracingEnabled)
+                {
+                    EventSourceTrace(string.Format(formatMessage, arg0), memberName);
+                }
+            }
+
+            private void EventSourceTrace<T1, T2, T3>(string formatMessage, T1 arg0, T2 arg1, T3 arg2, [CallerMemberName] string memberName = null)
+            {
+                if (EventSourceTracingEnabled)
+                {
+                    EventSourceTrace(string.Format(formatMessage, arg0, arg1, arg2), memberName);
+                }
+            }
+
+            private void EventSourceTrace(string message = null, [CallerMemberName] string memberName = null)
             {
-                CurlHandler.VerboseTrace(text, memberName, _easy);
+                CurlHandler.EventSourceTrace(message: message, memberName: memberName, easy: _easy);
```

Comment:
(minor): What's the advantage to using named arguments in this call? The variable names line up with the argument names.